bank_component.rb

```diff
@@ -0,0 +1,97 @@
+=begin
+ Feature 1: 
+ This feature will allow user to either sign up for an account.
+ Or log in with their last name,last 4 digits of their social security number, and password.        
+=end
```

Comment:
It's best practice to avoid block comments
https://github.com/rubocop/ruby-style-guide#rd-block-comments